scripts/make_universal.py

```
@@ -0,0 +1,215 @@
+	if VERBOSE:
+		joined = ",".join(matches)
+		print(f"Found {len(matches)} arches: {joined}");
+	return [*map(Arch, matches)]
+
+def run_make(path: str, args: list[str], additional_env=None):
+	if additional_env is None:
+		additional_env = {}
+	if VERBOSE:
+		print("Running: make " + " ".join(args))
+	env = os.environ.copy() | additional_env
+	p = Popen(["make", *args], stdout=PIPE, stderr=PIPE, env=env, cwd=path)
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)